Introduce point cloud expression filter dialog (#8456)
qgis/QGIS-Documentation: docs/user_manual/working_with_point_clouds/point_clouds.rst

  this process only if the CRS applied to the layer is a wrong
  one or if none was applied.

.. _pdal_querybuilder:

* :guilabel:`Provider Feature Filter`: allows to restrict data to use in the current project to a subset,
  using functions and expression at the PDAL data provider level.
  Press the :guilabel:`Query Builder` button at the bottom to start setting the filter.

  .. _figure_point_cloud_querybuilder:

  .. figure:: img/point_cloud_querybuilder.png
     :align: center

     Filtering point cloud layer to buildings at specific altitude

  .. _pdal_expressions:

  In the bottom part of the dialog, you can type a :guilabel:`Provider specific filter expression`.

>   Such expression should comply with the `PDAL expression specifications <https://pdal.io/en/latest/stages/expression.html>`_
>   and can be built thanks to :

Author: I suspect this is not true
```suggestion
  Such an expression can be built thanks to:
```
